bazel/repositories.bzl

```diff
@@ -5,8 +5,6 @@ load("@envoy_api//bazel:external_deps.bzl", "load_repository_locations")
 load(":repository_locations.bzl", "REPOSITORY_LOCATIONS_SPEC")
 load("@com_google_googleapis//:repository_rules.bzl", "switched_rules_by_language")
 
-PPC_SKIP_TARGETS = ["envoy.filters.http.lua"]
```

Comment:
I'm curious, assuming that diverging minimally from upstream is something we want: would initialising to an empty array here work and be preferable? It seems this whole PR could perhaps be reduced to this (plus perhaps a comment to annotate why we diverge here):

```bazel
PPC_SKIP_TARGETS = []
```